packages/clerk-js/package.json

```diff
@@ -118,7 +116,7 @@
     "files": [
       {
         "path": "./dist/clerk.browser.js",
-        "maxSize": "181kB"
+        "maxSize": "185kB"
```

Comment:
❓ How much is the actual increase?

Reply:
Seems to be about 4kb. We are at 184.66kb after the upgrade.